code/modules/modular_computers/computers/item/tablet_presets.dm

```diff
@@ -54,11 +54,15 @@
 
 /obj/item/modular_computer/tablet/preset/advanced/atmos/Initialize() //This will be defunct and will be replaced when NtOS PDAs are done
 	. = ..()
+	var/obj/item/computer_hardware/hard_drive/small/hard_drive = find_hardware_by_name("solid state drive")
 	install_component(new /obj/item/computer_hardware/sensorpackage)
+	hard_drive.store_file(new /datum/computer_file/program/alarm_monitor)
+	hard_drive.store_file(new /datum/computer_file/program/atmosscan)
 
 /obj/item/modular_computer/tablet/preset/advanced/engineering/Initialize()
 	. = ..()
 	var/obj/item/computer_hardware/hard_drive/small/hard_drive = find_hardware_by_name("solid state drive")
+	hard_drive.store_file(new /datum/computer_file/program/alarm_monitor)
 	hard_drive.store_file(new /datum/computer_file/program/supermatter_monitor)
```

Comment:
surely there's a better way to handle this

Reply:
It is how the other presets work but I tried to make it a list and to follow a few other examples from the code to make it cleaner and couldn't make it work.
Some didn't even throw errors but only the first item on the list would come pre installed.

Still, if you have any idea on how to make it work I can look more into it but it might be something that would need a small refactor on how the tablets work.

Reply:
it should just be a list of programs, but this is fine for now